ci-operator/step-registry/cucushift/installer/check/vsphere/zone/rhcos-template/cucushift-installer-check-vsphere-zone-rhcos-template-commands.sh

```diff
@@ -43,8 +47,28 @@ function check_rhcos() {
 	fi
 
     done
-
-    return ${check_result}
+    if [[ -f "${SHARED_DIR}/.openshift_install.log" ]]; then
+        echo "installation log found"
+        if [[ $(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains[*].topology.template" | wc -l) == "${#zones_name_from_config[@]}" ]];then
+	    if grep "Obtaining RHCOS image file" "${SHARED_DIR}"/.openshift_install.log; then
+              echo "Fail: template defined in install-config, should not import ova"
+	      check_result=1
+	    else
+	      echo "Pass: template defined in install-config, skip importing ova"
+	    fi
+        else
+	    if grep "Obtaining RHCOS image file" "${SHARED_DIR}"/.openshift_install.log; then
+              echo "Pass: not all templates defined in install-config, the log will print info about importing ova"
```

Comment:
When "not all templates defined in install-config, the log will print info about importing ova", the number of "Obtaining RHCOS image file" lines is the same as the number of  templates defined in install-config ?